### ISSUE TITLE
remreset warning about its obsolescence has typos

### ISSUE THREAD

**Issue by jfbu** [davidcarlisle/dpctex#18]
```
Package remreset Warning: The remreset package is obsolete:
(remreset)                \@removefomresset is defined.
                                  ^^^  ^^
```

**Comment by davidcarlisle** [comment]
I blame incompetence:-) I'll fix in the next release, thanks.

**Comment by davidcarlisle** [comment]
This was fixed by d7ef57a2a3d07b3d208a10dfa5670fc084e426a8
